svthalia/Reaxit
# Support events with slugs

Closes #384.

### Summary
Adds support for events with slugs as deeplink.
As a consequence of this EventCubit and RegistrationCubit have also been merged.

### How to test
Steps to test the changes you made:
1. Go to staging.thalia.nu/events/(some slug).
2. See that the event loads correctely.
3. If there are no registrations, it does not show an error.
